ner_v1/detectors/textual/name/name_detection.py

```diff
@@ -172,8 +234,7 @@ def replace_detected_text(self, text_detection_result):
                     ['my', 'name', 'is', 'yash', 'doshi']
 
         """
-
-        replaced_text = nltk_tokenizer.tokenize(self.text.lower())
+        replaced_text = self.text.lower().split()
```

Comment:
changes this to `self.text.lower().strip().split()`

Reply:
done